### PR TITLE
Remove two log messages, tweak logging with metrics, add log message for conductors

### PR DESCRIPTION
**Pull request by rabbah**
When logging only a metric, skip the message unless the log level is enabled.
Lowered log level on two messages.
Add log message for compositions.

@vvraskin can you review?
pg3/1873 🏃

### CODE REVIEW

**Comment by csantanapr** [comment]
@vvraskin bump

File: core/controller/src/main/scala/whisk/core/loadBalancer/ContainerPoolBalancer.scala [hunk 1]
```diff
@@ -189,7 +189,7 @@ class ContainerPoolBalancer(config: WhiskConfig, controllerInstance: InstanceId)
           this,
           start,
           s"posted to ${status.topic()}[${status.partition()}][${status.offset()}]",
-          logLevel = InfoLevel)
+          logLevel = DebugLevel)
```
Comment (vvraskin) [inline core/controller/src/main/scala/whisk/core/loadBalancer/ContainerPoolBalancer.scala]: You can skip this one, I think that debug is the default.

**Comment by csantanapr** [comment]
@rabbah you want to merge it as is, or address @vvraskin comment https://github.com/apache/incubator-openwhisk/pull/3299#discussion_r169305825 ?

**Comment by rabbah** [comment]
Explicit doesn’t hurt. I’d like to leave it.

**Comment by rabbah** [comment]
I made the change - it make the log command just one line.

**Pull request merged**